src/datacrumbs/server/process/event_processor.cpp

```diff
@@ -871,25 +871,20 @@ int main_process(int argc, char** argv, datacrumbs::EventProcessor* event_proces
 int main(int argc, char** argv);
 
 int main(int argc, char** argv) {
-  if (argc < 2) {
-    std::cerr << "Usage: " << argv[0] << " start|stop|run [args...]" << std::endl;
-    return 1;
-  }
-  std::string cmd = argv[1];
+  auto event_processor = datacrumbs::EventProcessor(argc, argv);
   std::string hostname = get_hostname();
   std::string timestamp = get_timestamp();
   std::string pidfile = "/tmp/datacrumbs_" + hostname + ".pid";
```

Comment:
This should be in `PREFIX_RUN`. Cmake will set it (could be `INSTALL_RUN`).

Reply:
DONE